src/test/java/com.breskul.bring/ApplicationContextImplTest.java

```diff
@@ -0,0 +1,54 @@
+package com.breskul.bring;
+
+import com.breskul.bring.dump.service.DbService;
+import com.breskul.bring.dump.service.PrinterService;
+import com.breskul.bring.exceptions.NoSuchBeanException;
+import com.breskul.bring.exceptions.NoUniqueBeanException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ApplicationContextImplTest {
+
+	private ApplicationContext applicationContextImplUnderTest;
+
+	@BeforeEach
+	void setUp() {
+		applicationContextImplUnderTest = new AnnotationConfigApplicationContext("com.breskul.bring.dump");
+	}
+
+	@Test
+	void testGetBeanShouldReturnBean() {
+		var result = applicationContextImplUnderTest.getBean(PrinterService.class);
+
+		assertNotNull(result);
+	}
+
+	@Test
+	void testGetBeanByNameShouldReturnBean() {
+		var result = applicationContextImplUnderTest.getBean("db", DbService.class);
+		assertNotNull(result);
```

Comment:
Tests `testGetBeanShouldReturnBean` and `testGetBeanByNameShouldReturnBean` only check that the method `getBean` return some object, but they don't check if that object is the correct one.

Reply:
It doesn't make sense to check if the object is correct. Because it's checked on a compilation level. 
`var` variable could confuse people here)

Reply:
It doesn't check that the `result` is an instance of `MySqlDbServiceImpl`. On a compilation level it only checks that result is not null and implements the `DbService` interface.

Reply:
yep, agree